easybuild/easyblocks/c/clang.py

```diff
@@ -67,6 +67,8 @@ def extra_options():
             'bootstrap': [True, "Bootstrap Clang using GCC", CUSTOM],
             'usepolly': [False, "Build Clang with polly", CUSTOM],
             'static_analyzer': [True, "Install the static analyser of Clang", CUSTOM],
+            # The sanitizer tests often fail on HPC systems due to the 'weird' environment.
+            'skip_sanitizer_tests': [False, "Do not run the sanitizer tests", CUSTOM],
```

Comment:
OK, fine.

Can you copy-paste the error messages in the PR description? I'll merge this in then.